README.md

```diff
@@ -17,7 +17,7 @@ wget -qO - https://raw.githubusercontent.com/xip-online-applications/xip-cli/mas
 Linux (AMD64):
 
 ```bash
-wget -qO - https://raw.githubusercontent.com/xip-online-applications/xip-cli/master/install/linux.sh | bash
+wget -qO - https://raw.githubusercontent.com/xip-online-applications/xip-cli/master/install/linux.sh | sudo bash
```

Comment:
Zou niet moeten, met name omdat de /usr/local/bin folder normaliter in het "bezit" moet zijn van de reguliere gebruiker.

Reply:
Linux is natuurlijk de uitzondering op de regel :) https://unix.stackexchange.com/questions/37724/permissions-ownership-of-usr-local-bin

Of misschien beter gezegd; in dit geval is mac de uitzondering.

Reply:
fresh install linux mint 20 heeft /usr/local/bin in bezit van root, en alles erin ook, maar wel readable voor andere gebruikers. Zie ook antwoord https://unix.stackexchange.com/questions/37724/permissions-ownership-of-usr-local-bin